### ISSUE TITLE
a bug when LogError

### ISSUE THREAD

**Issue by paulybtseng** [Xabaril/AspNetCore.Diagnostics.HealthChecks#1367]
**What happened**:
get unexpected log result and miss error information 

**What you expected to happen**:
get correct log result with error information

**How to reproduce it (as minimally and precisely as possible)**:
a exception occurs and is caught when send webhook request 

**Source code sample**:
In the class "HealthChecks.UI.Core.Notifications.WebHookFailureNotifier",
in the method "SendRequestAsync":
```
private async Task SendRequestAsync(Uri uri, string name, string payloadContent)
        {
            try
            {
                using var request = new HttpRequestMessage(HttpMethod.Post, uri)
                {
                    Content = new StringContent(payloadContent, Encoding.UTF8, Keys.DEFAULT_RESPONSE_CONTENT_TYPE)
                };
                using var response = await _httpClient.SendAsync(request, HttpCompletionOption.ResponseHeadersRead);
                if (!response.IsSuccessStatusCode)
                {
                    _logger.LogError("The webhook notification has not executed successfully for {name} webhook. The error code is {statuscode}.", name, response.StatusCode);
                }
            }
            catch (Exception exception)
            {
                **_logger.LogError($"The failure notification for {name} has not executed successfully.", exception);**
            }
        }
```
This sentence "_logger.LogError($"The failure notification for {name} has not executed successfully.", exception);" will result in just a string likes "The failure notification for some webhook name has not executed successfully."
And there is no exception information in the log.

**Comment by sungam3r** [comment]
PR is welcome.

**Comment by sungam3r** [comment]
@paulybtseng Thanks for report.
